base/notify-api/api-deployment.yaml

```diff
@@ -287,11 +287,11 @@ spec:
             httpGet:
               path: /_status?simple=true
               port: 6011
-            initialDelaySeconds: 15
+            initialDelaySeconds: 20
             periodSeconds: 3
             timeoutSeconds: 1
             successThreshold: 1
-            failureThreshold: 6
+            failureThreshold: 20
```

Comment:
We are total giving 1m20s of start up time to the app, with 20 seconds being the minimum initialization and then trying to get the ball rolling with a http test every 3 seconds, for 1 minute.

Reply:
Interesting!